enhancements/organizations/README.md

```
@@ -197,15 +210,62 @@ This flow uses standard OIDC protocol endpoints; OSAC does not provide custom to
    - Delete project via `DELETE /api/fulfillment/v1/organizations/{name}/projects/{name}` (with proper teardown, including nested projects if any)
    - List nested projects via `GET /api/fulfillment/v1/organizations/{name}/projects/{parent_name}/projects`
+   - Assign project permissions to users/groups via Keycloak Authorization Services
 
 5. Tenant Users can now provision resources within the Project (or nested Project) using Tenant APIs.
 
+#### Tenant User Workflow: Accessing Projects and Creating Resources
+
+1. Tenant User authenticates using the organization's IdP, receiving an OIDC token from Keycloak.
+
+2. Tenant User specifies the project context in API requests. The project is included in the request body or as a path parameter:
+   ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)